src/views/catalog/wizard/components/WizardDescriptionItem/WizardDescriptionItem.tsx

```diff
@@ -74,7 +76,11 @@ export const WizardDescriptionItem: React.FC<WizardDescriptionItemProps> = React
         );
       }
 
-      return <DescriptionListTerm>{titleWithCount}</DescriptionListTerm>;
+      return (
+        <DescriptionListTerm>
+          {titleWithCount} {label}
```

Comment:
Similarly as above, I'd recommend to add an extra space between the text and the label. Maybe adding it to the label component? Up to you...